src/ProfilesPage/ProfilePage.js

```diff
@@ -262,6 +291,21 @@ class ProfilePage extends Component {
             {this.renderEditUserButton()}
             {this.suspendUserButton()}
             {this.deleteUserButton()}
+            {deleteModalVisible && <Modal style={{borderRadius:"50px"}} open={this.state.deleteModalVisible} onClose={this.onCloseModal} center>
+          <h2 className='text-center'>Are You Sure?</h2>
+          <hr/>
+          <div className='container'>
+          <div className='row'>
+          <div className='col-md-6'>
+          <button className="btn btn-danger" onClick={this.onDelete}>Yup</button>
```

Comment:
Let's change these to just "Yes" and "No".

Reply:
I changed it.